### PR TITLE
Display ids as decimal, not hexa

### PR DESCRIPTION
**Pull request by rakoo**
Hey there, thanks a lot for your project it's a great help !

I noticed in logs that the ids are funky, sometimes referring to "a" even though it's supposed to be a `uint64`. This is because the format was wrong. Here's a fix

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,11 +14,11 @@ clean:
 cover: clean 
 	mkdir ${PWD}/cover 
 	go clean -testcache
-	GOFLAGS=-mod=vendor go test `go list ./... | grep -v github.com/shaj13/raft/rafttest` -timeout 30s -race -v -cover -coverprofile=${PWD}/cover/coverage.out
+	GOFLAGS=-mod=vendor go test `go list ./... | grep -v github.com/rakoo/raft/rafttest` -timeout 30s -race -v -cover -coverprofile=${PWD}/cover/coverage.out
 
 rafttest: clean
 	go clean -testcache
-	GOFLAGS=-mod=vendor go test github.com/shaj13/raft/rafttest -race 
+	GOFLAGS=-mod=vendor go test github.com/rakoo/raft/rafttest -race
```
Comment (Tochemey) [inline Makefile:21]: Why are you making this change?

**Review by Tochemey — CHANGES_REQUESTED** [review]
I think your pr is not supposed to change package path otherwise you will break existing code. I suggest you make your fix without changing the packages paths. Thank you

File: controller.go [hunk 2]
```diff
@@ -6,11 +6,11 @@ import (
 	"io"
 	"sync"
 
-	"github.com/shaj13/raft/internal/membership"
-	"github.com/shaj13/raft/internal/raftengine"
-	"github.com/shaj13/raft/internal/raftpb"
-	"github.com/shaj13/raft/internal/storage"
-	"github.com/shaj13/raft/internal/transport"
+	"github.com/rakoo/raft/internal/membership"
```
Comment (Tochemey) [inline controller.go:9]: Why this change?

File: internal/membership/pool_test.go [hunk 3]
```diff
@@ -6,9 +6,9 @@ import (
 	"time"
 
 	"github.com/golang/mock/gomock"
-	"github.com/shaj13/raft/internal/raftpb"
-	"github.com/shaj13/raft/internal/transport"
-	"github.com/shaj13/raft/raftlog"
+	"github.com/rakoo/raft/internal/raftpb"
```
Comment (Tochemey) [inline internal/membership/pool_test.go:9]: Please revert these changes otherwise I am afraid this PR will not be merged.

**Review by Tochemey — CHANGES_REQUESTED** [review]

**Comment by rakoo** [comment]
Oops, sorry, I made a mistake pushing the path change. Reopened at !20

**Pull request closed without merging**